package.json

```diff
@@ -178,6 +178,7 @@
     "@fastify/ajv-compiler": "^1.0.0",
     "@fastify/proxy-addr": "^3.0.0",
     "abstract-logging": "^2.0.0",
+    "ajv": "^6.12.2",
```

Comment:
Can we fix the issues without re-adding this dependency? If not for v3 then for v4?

Reply:
I think so but it will require some reorg of our types.

Reply:
We should export `ajv` types from `@fastify/ajv-compiler`, and we can have a the appropriate types for `ajv`.
Let say `@fastify/ajv-compiler@1` will export `ajv@6`, `@fastify/ajv-compiler@1` will export `ajv@8`.

Reply:
Could you send a PR there or guide me on how to do that?

Reply:
@Eomm https://github.com/fastify/ajv-compiler/pull/12